<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" />
  <arg name="load_gripper" default="true" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find serl_franka_controllers)/config/serl_franka_controllers.yaml" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="true" output="screen" args="cartesian_impedance_controller"/>
</launch>
